docs/src/main/asciidoc/kogito-guide.adoc

```diff
@@ -0,0 +1,306 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+include::./attributes.adoc[]
+= {project-name} - Using Kogito to add business automation capabilities to an application
+
+This guide demonstrates how your {project-name} application can use Kogito to add business automation
+to power it up with business processes and rules.
+
+Kogito is a next generation business automation toolkit that originates from well know Open Source projects
+Drools (for business rules) and jBPM (for business processes). Kogito aims at providing another approach
+to business automation where the main message is to expose you business knowledge (processes, rules and decisions)
+in domain specific way.
+
+You can find more information about Kogito in https://kogito.kie.org/[Kogito website].
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE (preferred Eclipse with BPMN modeller plugin)
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+* Docker
+
+=== Install Eclipse with modelling plugins
+
+To be able to make use of visual modelling of your processes download Eclipse IDE and
+install from Market place
+
+* Eclipse BPMN2 Modeler plugin (with jBPM Runtime Extension)
+
+== Architecture
+
+In this example, we build a very simple microservice which offers one REST endpoint:
+
+* `/persons`
+
+This endpoint will be automatically generated based on business process, that in turn will
+make use of business rules to make certain decisions based on the data being processed by the process.
```

Comment:
Maybe just `being processed`? `being processed by the process` is a bit redundant.